### ISSUE TITLE
Unknown Conda Environment in Zero to Hero README.md

### ISSUE THREAD

**Issue by JeffreyLind3** [meta-llama/llama-stack#544]
In the Zero to Hero README.md, there exists the following step for "Install Dependencies and Set Up Environment":

4. **Install Llama Stack**:
   - Open a new terminal and install `llama-stack`:
     ```bash
     conda activate hack
     pip install llama-stack==0.0.53
     ```

However, the "hack" conda environment was never created or referenced in this documentation, nor anywhere else that I can find in the repository. I *could* be missing something, but not sure. If the user is supposed to create the hack environment, the docs ideally should state this.

**Comment by JeffreyLind3** [comment]
Update: 

I found a commit (9928405e2cf4689f9e377d8cf3146aed15849e04) where the hack environment was changed to be called "ollama" in the README, but it seems that the hack environment was not changed in the step above. I can submit PR if this is the issue.

**Comment by JeffreyLind3** [comment]
Fixed by #552
